deployments/deployments.json

```diff
@@ -23,12 +23,13 @@
     "RateModelStore": { "address": "0x18a200A2427e243154EC8217890Bc08062afc5A2", "blockNumber": 10468968 }
   },
   "5": {
-    "AcrossConfigStore": { "address": "0x3215e3C91f87081757d0c41EF0CB77738123Be83", "blockNumber": 6860550 },
```

Comment:
I don't think you needed to redeploy any existing contracts right? Only deploy the new MerkleDistributor?

Reply:
Yea I didn't redeploy these, only the `MerkleDistributor`. But it seems that the artifacts within `deployments/goerli` were updated but the `deployments/deployments.json` didn't reflect these updates. When I initially ran `yarn`, it automatically updated this file.

Reply:
What deploy command did you use? Ideally this PR should only add a single line for merkle distributor on goerli